R/testing_utils.R

```diff
@@ -0,0 +1,64 @@
+
+
+# [Title]   Build A Library For Testing
+# [DESC]    Loads all packages necessary for testing into a another directory, 
+#           preferably a temporary directory. This function also confirms successful installation. 
+# [param]   currentLibPath (string) path to the current library in which pkgnet resides
+# [param]   targetLibPath (string) path to the location of the new directory
+# [return]  boolean TRUE
+.BuildTestLib <- function(currentLibPath
+                         , targetLibPath){
+    
+    # packages to be built
+    pkgList <- list(baseballstats = system.file('baseballstats'
+                                                , package = "pkgnet"
+                                                , lib.loc = currentLibPath
+                                                )
+                    , sartre = system.file('sartre'
+                                           , package = "pkgnet"
+                                           , lib.loc = currentLibPath
+                                           )
+                    , pkgnet = find.package(package = 'pkgnet'
+                                   , lib.loc = currentLibPath
+                                   )
+                    )
+    
+    installResult <- sapply(X = names(pkgList)
```

Comment:
this would be clearer if you ended this line with `sapply(` and did all the indentation starting four spaces over from the `installResult` (for this line and all the others)